ONSdigital/ssdc-rm-notify-service
# Differentiate between fulfilments and action rule sends in event log

# Motivation and Context

Emails and sms from fulfilments and action rules are recorded the same in the event log. We want to be able to differentiate between these.
# What has changed



Mostly renames of dtos and queues to confirmation instead of fulfilment specific.
Scheduled flag passed through to differentiate email and sms between fulfilments and action rule.
# How to test?



Build and make up with everything else on the card.
Run ATs
Check the emails and sms sent using action rules are recorded in the event log with the types ACTION_RULE_SMS_CONFIRMATION/ACTION_RULE_EMAIL_CONFIRMATION.
# Links



https://trello.com/c/icUNfLJQ/3057-can-we-differentiate-between-fulfilment-requests-and-action-rule-sends-in-event-log-for-sms-email-8
# Screenshots (if appropriate):
